system-int-test/src/cmr/system_int_test/utils/variable_util.clj

```diff
@@ -169,30 +172,27 @@
                (assoc :metadata "" :deleted true))
            (dissoc concept :revision-date :created-at :transaction-id :extra-fields)))))
 
-(defn sort-expected-variables
-  "Sorts the variables using the expected default sort key."
-  [variables]
-  (sort-by identity
-           (fn [t1 t2]
-             (compare (:variable-name t1) (:variable-name t2)))
-           variables))
-
 (def variable-names-in-expected-response
-  [:concept-id :revision-id :variable-name :description :originator-id])
+  [:concept-id :revision-id :provider-id :native-id])
 
 (defn assert-variable-search
   "Verifies the variable search results"
   ([variables response]
    (assert-variable-search nil variables response))
   ([expected-hits variables response]
    (let [expected-items (->> variables
-                             sort-expected-variables
-                             (map #(select-keys % variable-names-in-expected-response)))
+                             (map #(select-keys % variable-names-in-expected-response))
+                             seq
+                             set)
```

Comment:
Sorting is not part of this PR even though I threw in the sorting code on variable_name and provider_id. My intention is to do sorting in a separate issue. I have tried to include sorting in this PR, but it turns out to be a little bit trickier than I thought because I don't have variable_name back from the ingest response which we probably should add to the ingest endpoint. Because it is tricky to add sorting, I didn't include the sorting test in this PR. As you can see I am intentionally dissocing variable_name and measurement in the assertion function. These will be changed when we work on the sorting issue.